src/lib/components/EventStore/index.js

```diff
@@ -80,12 +83,19 @@ class EventStore extends HTMLElement {
         nextPendingDay = day;
       }
 
-      // Is this day actually active (within exact time range)?
-      const isActive = now >= bufferStart && now <= bufferEnd;
+      // Is this day active (within the buffer time range)?
+      const isActive = now >= bufferStart && now < bufferEnd;
       if (isActive && this._activeDay !== day) {
         this._activeDay = day;
         change = true;
       }
+
+      // Is this the active day for chat (within the actual time range)?
+      const isActiveChat = now >= actualStart && now < actualEnd;
+      if (isActiveChat && this._activeChatDay !== day) {
+        this._activeChatDay = day;
```

Comment:
Thinking through this...

I don't think I need to know the activeChatDay, I think I only need to know if chat is enabled or not. So if we are within the bounds of the actual day then chat is enabled, and if not it's disabled.